x/protorev/genesis.go

```diff
@@ -43,8 +43,15 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	k.SetPoolWeights(ctx, poolWeights)
 
 	// Configure the initial base denoms used for cyclic route building
-	baseDenomPriorities := []string{types.OsmosisDenomination, types.AtomDenomination}
-	k.SetBaseDenoms(ctx, baseDenomPriorities)
+	baseDenomPriorities := []*types.BaseDenom{
+		{
+			Denom:    types.OsmosisDenomination,
+			StepSize: sdk.NewInt(1_000_000),
+		},
+	}
```

Comment:
Maybe I'm misunderstanding the word "priorities", shouldn't this read from genesis state?

Reply:
(probably best resolved in a followup PR)